articles/storage/blobs/storage-dotnet-how-to-use-blobs.md

```diff
@@ -20,385 +20,370 @@ ms.author: tamram
+    "Block blob with name 'Forest-river-wallpaper-Hd.jpg', content type 'application/octet-stream', size '2690689', and URI 'http://127.0.0.1:10000/devstoreaccount1/quickstart/Forest-river-wallpaper-Hd.jpg'",
+    "Block blob with name 'Grand-Canyon-Colorado-River.jpg', content type 'application/octet-stream', size '261788', and URI 'http://127.0.0.1:10000/devstoreaccount1/quickstart/Grand-Canyon-Colorado-River.jpg'",
+    "Block blob with name 'Smith-River.jpg', content type 'application/octet-stream', size '519207', and URI 'http://127.0.0.1:10000/devstoreaccount1/quickstart/Smith-River.jpg'",
+    "Block blob with name 'grand-canyon-colorado-river-1.jpg', content type 'application/octet-stream', size '232356', and URI 'http://127.0.0.1:10000/devstoreaccount1/quickstart/grand-canyon-colorado-river-1.jpg'",
+    "Block blob with name 'ws_Columbia_Mountain_River_Grass_1366x768.jpg', content type 'application/octet-stream', size '450769', and URI 'http://127.0.0.1:10000/devstoreaccount1/quickstart/ws_Columbia_Mountain_River_Grass_1366x768.jpg'"
+]
 ```
 
-Anyone on the Internet can see blobs in a public container. However, you can modify or delete them only if you have the appropriate account access key or a shared access signature.
+## Basic blob operations
 
-## Upload a blob into a container
-Azure Blob Storage supports block blobs and page blobs.  In most cases, block blob is the recommended type to use.
+The following sections will walk through the basics of interacting with blob storage within an ASP.NET Web API project:
 
-To upload a file to a block blob, get a container reference and use it to get a block blob reference. Once you have a blob reference, you can upload any stream of data to it by calling the **UploadFromStream** method. This operation creates the blob if it didn't previously exist, or overwrites it if it does exist.
+* Storage account and container creation, done in a constructor.
+* Uploading a file on the server to a blob.
+* Download a file from a blob to the server.
+* Deleting a blob.
+* Listing metadata about each blob.
 
-The following example shows how to upload a blob into a container and assumes that the container was already created.
+These five basic operations should be enough for you to do a large amount of tasks with Blob Storage.
 
-```csharp
-// Retrieve storage account from connection string.
-CloudStorageAccount storageAccount = CloudStorageAccount.Parse(
-    CloudConfigurationManager.GetSetting("StorageConnectionString"));
+## Create a storage account and container
 
-// Create the blob client.
-CloudBlobClient blobClient = storageAccount.CreateCloudBlobClient();
+In the constructor, a [Storage Account](https://docs.microsoft.com/dotnet/api/microsoft.windowsazure.storage.cloudstorageaccount?view=azure-dotnet) and [Container](https://docs.microsoft.com/dotnet/api/microsoft.windowsazure.storage.blob.cloudblobcontainer?view=azure-dotnet) are created as follows:
```

Comment:
I think the phrasing is a bit off to me here. I would say:

The first thing that we need to do is setup our Storage Account and Container. This can be initialized in the constructor of the class. (or where is it being constructed?)

Reply:
Good feedback. I'll reword it like that.